homeassistant/helpers/template.py

```diff
@@ -1262,7 +1262,12 @@ class TemplateEnvironment(ImmutableSandboxedEnvironment):
 
     def __init__(self, hass):
         """Initialise template environment."""
-        super().__init__()
+        if hass:
+            super().__init__(
+                loader=jinja2.FileSystemLoader(hass.config.config_dir + "templates")
```

Comment:
why did you tag me in the conversation?

Reply:
Sorry, was trying to say that @ raman on HA discord helped me and wanted to thank them.  But of course that could be someone completely different.  My deepest, sincerest apologies for tagging you.

Reply:
I meant to tag @raman325